src/nade/components/NadeActions/NadeReportButton.tsx

```
@@ -23,14 +23,14 @@ export const NadeReportButton: FC<Props> = ({ nadeId }) => {
       nadeId,
       message: reportMsg,
     };
-    ReportApi.add(report);
+    ReportApi.add(report).then();
```

Are these needed?
Is it related to the TS2488 errors you are getting?

Otherwise you can remove it.
I will later make this more robust, as at the moment It's ignoring any errors.

---

nope, they are not needed... 

> I will later make this more robust, as at the moment It's ignoring any errors.

perfect!